### ISSUE TITLE
Add support for remote device proxying in iPad/iPhone app

### ISSUE THREAD

**Issue by sushant40** [ProxymanApp/Proxyman#859]
I'd like to be able to proxy traffic from my iPhone on my iPad (running proxyman). Is this possible?

**Comment by NghiaTranUIT** [comment]
@sushant40 yes, it’s possible. Please check out this guideline https://docs.proxyman.io/debug-devices/ios-device
